scbctl/cmd/scans.go

```diff
@@ -116,6 +126,74 @@ func NewScanCommand() *cobra.Command {
 	}
 
 	scanCmd.Flags().String("name", "", "Name of the created scan. If no name is provided, the ScanType will be used as the name")
+	scanCmd.Flags().Bool("follow", false, "Follow the log output")
 
 	return scanCmd
 }
+
+func followScanLogs(ctx context.Context, kubeclient client.Client, namespace, scanName string) error {
+	// Find the job associated with the scan
+	jobList := &batchv1.JobList{}
+	labelSelector := client.MatchingLabels{}
```

Comment:
The scan job all a label on them.


```suggestion
	labelSelector := client.MatchingLabels{
        "securecodebox.io/job-type": "scanner"
    }
```

This way it wouldn't have to go through the whole list of jobs, but only scan jobs.
Which also ties in nicely witht he other suggestion below (owner reference) because this way you know for sure that it is the scanner job of the scan.